index.html

```diff
@@ -15,19 +15,19 @@
   </head>
   <body>
     <header>
-      <div class="mobile-bar">
-        <li class="time"><img src="assets/Bars/Status Bars/↳ Timemobile-top.png"></li>
-        <li class="cellular"><img src="assets/Bars/Status Bars/Cellular Connectionmobile-top.png"></li>
-        <li class="wifi"><img src="assets/Wifiwifi.png"></li>
-        <li class="battery"><img src="assets/Bars/Status Bars/Batterymobile-top.png"></li>
-      </div>
+      <nav class="nav">
+       <a href="#"><h1 class="my-logo">My logo</h1></a>
```

Comment:
- [Optional] Great job for adding `nav`, I think the logo is not part of the navigation so make the `a` logo outside the `nav` tag. there will be additional navigation which you will add later 👍👍